tests/_test_mscolab/test_server.py

```diff
@@ -448,6 +483,8 @@ def _save_content(self, operation, userdata=None):
         if userdata is None:
             userdata = self.userdata
         user = get_user(userdata[0])
+        self.fm.save_file(operation.id, XML_CONTENT1, user)
+        return user
         fm = FileManager(self.app.config["MSCOLAB_DATA_DIR"])
```

Comment:
Please check this. This looks not correct, the lines after the return can't be reached